lib/runtime/test_helpers.go

```diff
@@ -85,7 +87,18 @@ func GetRuntimeBlob(testRuntimeFilePath, testRuntimeURL string) error {
 		return nil
 	}
 
-	resp, err := http.Get(testRuntimeURL)
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	req, err := http.NewRequestWithContext(ctx, http.MethodGet, testRuntimeURL, nil)
```

Comment:
nit nit I'm going to sound annoying, but you could actually use `context.WithTimeout(context.Background(), someTimeout)` and the default http client 😄  But the current implementation is good too 👍!